doc/source_interpreter.tex

```diff
@@ -11,4 +9,139 @@ \section*{Interpreter Support}
 }
 apply_in_underlying_javascript(times, list(2, 3)); // returns 6
 \end{lstlisting}
+\item \lstinline{parse(x)}: \textit{primitive}, returns the parse tree that results from parsing
+the string \lstinline{x} as a Source program. The following two pages describe the shape of the parse tree.
 \end{itemize}
+
+\newpage
+\KOMAoptions{paper=landscape,pagesize}
+\recalctypearea
+
+\begin{alignat*}{9}
+&& \textit{program}    &&\quad ::= &\quad && \textit{statement} \ \ldots
+                                                           && \texttt{list("sequence", list of <statement>)} \\[1mm]
```

Comment:
Exception if the program is a single statement, in which case it should just be `<statement>`.

Interestingly, if the program is empty, then it's a sequence containing an empty list.

Reply:
Yes, that's an optimization to make the parse tree more readable. I'll mention it in a footnote.